test/input/push-ember-object.js

```diff
@@ -0,0 +1,15 @@
+import Ember from 'ember';
+
+export default Ember.Component.extend({
+  pollMultiOptions: Ember.computed('someArray.[]', function() {
+    const options = Ember.A([]);
+
+    this.get('someArray').forEach(function(option) {
+      options.push(Ember.Object.create({
+        content: option.get('content'),
+      }));
+    });
+
+    return options;
+  })
```

Comment:
This is some really old code and should probably be written as a map (or reconsidered entirely).